Makefile

```diff
@@ -105,40 +107,30 @@ proxy: $(shell find $(proxy) $(find_options))
 wallet-prod: wallet-node-modules $(shell find $(wallet)/src $(find_options))
 	$(log_start)
 	$(docker_run_in_wallet) "rm -f .env && cp ops/prod.env .env"
```

Comment:
Switching to `rm -rf` should make switching from other branches easier w/out causing any other problems right? But I see this change was reverted in commit 4f713f9, was that bc it caused some problems?